docs/adr/0001-provide-mechanism-to-show-hide-sections-of-accessibility-statement.md

```diff
@@ -0,0 +1,111 @@
+
+## Decision Outcome
+
+Chosen option: Pending.
+
+### Positive Consequences <!-- optional -->
+
+* We will be able to cater to the VOA team's that require different content on the accessibility statement who can't currently update or change content. 
+* The VOA team's will not try host their own accessibility statements.
+* The VOA team's will not waste time in being mistakenly contacted because of incorrect content on there accessibility statement.
+
+### Negative Consequences <!-- optional -->
+
+* Changes could impact on current service yaml files, which may lead to writing scripts to bulk update the service yaml files to use the new model.
+* Team's who currently use the accessibility statement frontend may be required to change the way they create service yaml files going forward, which may have slow uptake or cause more support queries for PlatUI.
+
+## Pros and Cons of the Options <!-- optional -->
+
+### [option 1]
+
+* Good, because no further work required.
+* Bad, because it doesn't solve the problem.
+
+### [option 2]
+
+`AboutThisService.scala` fragment would need to be updated so that it can take a new property for displaying VOA specific content. Currently, the section of content is static and comes directly from the internationalised messages property file with key `using-service.this-service-or-app-is-run`
+
+`ContactInformation.scala` fragment would need to be updated to take a flag property and the content below
+```html
+<p class="govuk-body">@{messages("in-person-contact.text-relay")}</p>
+<p class="govuk-body">@{messages("in-person-contact.bsl")}</p>
+```
+would need to either be displayed or not dependent on that new flag property.
+
+* Good, because the change would not be breaking to existing service yaml files as we are introducing new properties and not changing the existing view model dramatically.
+* Bad, because it is a less flexible approach, and means other team's in the future who need to make changes, will introduce multiple different properties that would eventually become confusing and less maintainable.
+
+### [option 3]
+
+Example of a new accessibility statement view model, this would introduce view models specific for each fragment within the accessibility statement.
+```scala
+case class AccessibilityStatement(
+  // common properties specific to the accessibility statement can be on the root level independently 
+  mobilePlatform: Option[MobilePlatform],
+  aboutThisService: Option[AboutThisService] = None,
+  complaintsProcess: Option[ComplaintsProcess] = None,
+  contactInformation: Option[ContactInformation] = None,
+  howAccessibleIsThisService: Option[HowAccessibleIsThisService] = None,
+  howServiceWasTested: Option[HowServiceWasTested] = None,
+  howToGetExtraSupport: Option[HowToGetExtraSupport] = None,
+  introduction: Option[Introduction] = None,
+  reportingAccessibilityProblems: Option[ReportingAccessibilityProblems] = None,
+  technicalDetails: Option[TechnicalDetails] = None
+) 
```

Comment:
Would this represent a totally blank accessibility statement, or would it be a default one as we have now with no overrides?

Reply:
@oscarduignan Good question, agree with this point.

Reply:
@oscarduignan very good question, my thought was and I should include this into the ADR. When teams create new service yaml files they would use a default template that would include all the fragments by default and from that point they can update the template as need be. Not sure if this may be too error-prone as an approach.